pkg/app/info.go

```diff
@@ -162,6 +162,7 @@ type Data struct {
 func (d *Data) MarshalLog(addField func(key string, v interface{})) {
 	if d.Name != "unknown" {
 		addField("app.name", d.Name)
+		addField("service_name", d.Name)
```

Comment:
should it be `service.name` ?

Reply:
@artemchernyak-outreach +1 for consistency + ticket for deprecating app.name ?